overlap-scripts/kubectl-kubestellar-ensure-wmw

```diff
@@ -77,19 +77,33 @@ then kubectl ws "${kubectl_flags[@]}" "$wmw_name"
 else kubectl ws "${kubectl_flags[@]}" create "$wmw_name" --enter
 fi
 
-if ! kubectl "${kubectl_flags[@]}" get APIBinding bind-espw &> /dev/null; then
-kubectl "${kubectl_flags[@]}" apply -f - <<EOF
-apiVersion: apis.kcp.io/v1alpha1
-kind: APIBinding
-metadata:
-  name: bind-espw
-spec:
-  reference:
-    export:
-      path: root:espw
-      name: edge.kubestellar.io
-EOF
-fi
+function run_kb_konnector_for_wmw() {
+    if kubectl get ns kube-system &> /dev/null; then
+        echo "namespace kube-system already exists, skipping"
+    else
+        echo "creating namespace kube-system"
+        kubectl create ns kube-system
+    fi
+    echo "starting kube-bind konnector for wmw in background"
+    konnector &> /dev/null &
+    if ! pgrep konnector &> /dev/null; then
```

Comment:
Don't you want to check for the _right_ `konnector` process ?

Reply:
Yes. The code here is moved to the `kubestellar-kube-bind` script [here](https://github.com/kubestellar/kubestellar/blob/c1c97cdd50c2234aadaaf41b299a8568529e458c/scripts/inner/kubestellar-kube-bind#L96-L99). We will use #1337 to track the issue.